redisvl/extensions/llmcache/semantic.py

```diff
@@ -333,19 +366,66 @@ def store(
         self._check_vector_dims(vector)
 
         # Construct semantic cache payload
+        now = time()
```

Comment:
We should use `datetime.utcnow()` to generate a UTC tz and then use the numeric version of that. Let's also make this a utility function that can be imported from `redisvl.utils` module. We will use this for multiple things across extensions.